### DISCUSSION TITLE
I just want to thank you for the template

### DISCUSSION THREAD

**Discussion by berkcomba** [seapagan/fastapi-template#610] [General]
Well, I have been using this template for quite some time now, and I have to say that it is by far the best template I have ever experienced. I just want to thank the contributors for building such a time-saving resource. FastAPI, unlike other frameworks, is not quite popular, and sometimes it can be hard to find resources and you need to do everything from scratch. But, hardworking people like you have made it more enjoyable.

I will surely contribute when my workload decreases.

Thanks a lot!

**Comment by seapagan** [comment]
@berkcomba Thank you for the kind words :grin: 

Glad you are getting good usage out of the template, that makes it all worth-while for me. 

Keep watching as I plan to get back to work on it in the next few weeks and add more functionality, plus proper documentation to make it easier to use and cusomize.
